scripts/options.js

```diff
@@ -15,17 +15,24 @@
       Title: '.playerBarSong',
       Cover: '.playerBarArt',
       Playtime: '.elapsedTime'
+    },
+    spotify: {
```

Comment:
Maybe we should allow someone to auth their requests if desired. IDK how this scales due to rate limiting for unauth requests.